rkt/run.go

```diff
@@ -103,6 +103,7 @@ func addAppFlags(cmd *cobra.Command) {
 	cmd.Flags().Var((*appWorkingDir)(&rktApps), "working-dir", "override the working directory of the preceding image")
 	cmd.Flags().Var((*appReadOnlyRootFS)(&rktApps), "readonly-rootfs", "if set, the app's rootfs will be mounted read-only")
 	cmd.Flags().Var((*appMount)(&rktApps), "mount", "mount point binding a volume to a path within an app")
+	cmd.Flags().Var((*appExpose)(&rktApps), "expose", "expose ports of the application (example: '--expose=http,protocol=tcp,port=8080')")
```

Comment:
This new flag should be documented in `run.md`, and possibly also also to `prepare` / `run-prepared` (I didn't check, I think this also adds it to the command so maybe only the doc part is missing).